src/components/pages/TodoPage.js

```diff
@@ -24,9 +24,28 @@ export default class TodoPage extends Component {
                 Completed
               </th>
             </tr>
+            <tr>
+              <th>
+
+              </th>
+              <th>
+
+              </th>
+              <th>
+
+              </th>
+            </tr>
           </thead>
         </table>
       </div>
     )
   }
 }
+
+const mapStateToProps = state => {
+  return {
+    todos: state.todos.toJS()
+  }
+}
+
+export default connect(mapStateToProps)(TodoPage)
```

Comment:
use 
```javascript
export default mastermind.connectStore(TodoPags, ['todos'])
```

connect is fine, but use mastermind to connect to the store

Reply:
do i still need the const mapStateToProps chunk (45-49)? Or is it just the export default connect..... statement now because I am getting errors surrounding the toJS method but it's not in my TodoPage.js file anymore

Reply:
nope, no longer needed, they're both valid, but only use one of them

Reply:
<img width="1034" alt="screen shot 2018-01-26 at 3 42 37 pm" src="https://user-images.githubusercontent.com/33434613/35459793-9cca142a-02af-11e8-8ed3-afb73661e3ed.png">
Can you take a look at this error message i'm getting on the local host then? i don't use toJS but if it's underlying in the mastermind.connectStore method then i don't know what to do?

Reply:
```
// @flow

import React, { Component } from 'react'
import mastermind from './../../mastermind'


class TodoPage extends Component {

  render() {

    return (
      <div>
        <h1 className="placeholder-page-label"> Todo Page </h1>
        <table className="ui celled striped table todos-table">
          <thead>
            <tr>
              <th>
                Title
              </th>
              <th>
                Description
              </th>
              <th>
                Completed
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>
                Get weird
              </td>
              <td>
                Let's get weird
              </td>
              <td>

              </td>
            </tr>
          </tbody>
        </table>
      </div>
    )
  }
}

export default mastermind.connectStore(TodoPage, ['todos'])
```

also i've only made changes to the TodoPage.js i dont know if i have to change any other files so that this connection can be made?

Reply:
you need to add the the todos branch in ```initialStoreState```

make it an empty object